README.md

```diff
@@ -1,3 +1,24 @@
+# Deprecation Notice
+
+This project is no longer receiving updates. The Aries Toolbox was an
+invaluable tool in helping to bootstrap the Aries Ecosystem by providing ways to
+interact with Agents in development. It was one of the earliest UIs to an agent
+and it helped shape our thinking of Agents, DIDComm, AnonCreds, and more.
+
+While it served a critical role in the early days of the Aries project, the
+Aries Toolbox has worked itself out of a job. Today, we have a rich ecosystem
+of agents. It is no longer necessary to demonstrate DIDComm and Credential
+Issuance and Verification over DIDComm using the Toolbox because we can do it
+with any number of available Issuer, Verifier, and Holder agents.
+
+There are some components of the Toolbox that remain relevant and valuable
+tools during Agent development, such as live composition of DIDComm messages
+and being able to analyze and inspect DIDComm messages in plaintext. These
+concepts will live on in other tools and projects.
+
+If you have questions about this project or replacements for this project, feel
+free to reach out on Hyperledger's Discord: https://discord.gg/hyperledger
```

Comment:
Would it be beneficial to add alternatives here?

I know someone (anonyome?) built an UI on top of ACA-Py and there's definitely more in this category. We have Siera Desktop (https://github.com/animo/siera-desktop) which also replaces some parts of the toolbox.

Reply:
Definitely — please add the alternatives that are available.

Reply:
My only concern is keeping the alternatives list current, which we can't do if it's archived. We could include "at the time of archival, here are options. Check with the community for other alternatives." as a way to address the impending staleness.